figure_generation/nbconverted/figure_generation.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Figure generation
+
+# In[1]:
+
+
+from IPython.display import Image, display, SVG
+import svgutils.transform as sg
+import numpy as np
+from lxml import etree
+import os
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+
+
+# In[2]:
+
+
+# Directory of output figures
+output_directory = "output/"
+if not os.path.exists(output_directory):
+    os.makedirs(output_directory)
+
+
+# ## Function to plot
+
+# In[3]:
+
+
+def make_figure_panel(filename, scale_x_input, scale_y_input, x_loc, y_loc):
+    panel = (
+        sg.fromfile(filename)
+    )
+    
+    panel_size = (
+        np.round(float(panel.root.attrib['width'][:-2])*1.33, 0), 
+        np.round(float(panel.root.attrib['height'][:-2])*1.33, 0)
+    )
+
+    scale_x = scale_x_input
+    scale_y = scale_y_input
+
+    print(f"original: {panel_size}")
+    print(f"scaled:{(panel_size[0]*scale_x,panel_size[1]*scale_y)}")
+
+    panel = panel.getroot()
+    panel.scale_xy(x=scale_x, y=scale_y)
+    panel.moveto(x_loc, y_loc)
+    
+    return panel
+
+
+# ## Figure 1
+
+# In[4]:
+
+
+# Create panels for figure 1
+panel_1a = make_figure_panel(
+    "fig1A.svg",
+    2.5, 
+    2.5,
+    30,
+    10
+)
+panel_1b = make_figure_panel(
+    "../human_general_analysis/logs/NN_2500_30/tybalt_2layer_30latent_hist.svg",
```

Comment:
Would suggest integrating Pathlib, but not necessary for this PR.

Reply:
Very much agree. I really like Pathlib -- a much cleaner way to set parameters. Something for the next project!